koji_wrapper/factory.py

```diff
@@ -0,0 +1,48 @@
+
+from os import path
+
+import koji
+from koji_wrapper.exceptions import UnknownAuthMethod
+
+
+class KojiWrapperFactory(object):
+
+    @classmethod
+    def load_profile(cls, profile='koji', user_config=None):
```

Comment:
Is there a case where we need to call this directly?  I am not sure what it would be, if so, can you tell me?  If there is not a need for the user to call this, I would propose not making it a class method, and also making it a non_public method that open_session can just call

Reply:
I personally would prefer atomic building block wrappers that you can inspect and work with but if you have strong preferences it can be rotated around other ways.